tests/components/snoo/test_config_flow.py

```diff
@@ -73,18 +73,17 @@ async def test_form_auth_issues(
         },
     )
     # Reset auth back to the original
-    bypass_api.set_auth_error(None)
     assert result["type"] == FlowResultType.FORM
     assert result["errors"] == {"base": error_msg}
-
+    bypass_api.authorize.side_effect = None
+    bypass_api.authorize.return_value = MOCKED_AUTH
```

Comment:
```suggestion
```

Reply:
this is already set in the fixture right? Setting a side_effect doesn't overwrite the return_value so this shouldn't be needed